app/src/main/res/values/strings.xml

```diff
@@ -2,6 +2,10 @@
     <string name="app_name">Presence Publisher</string>
     <string name="host_title">MQTT Host</string>
     <string name="host_summary">IP address or hostname.%nCurrent value: %s</string>
+    <string name="login_title">MQTT Login</string>
+    <string name="login_summary">User Login.%nCurrent value: %s</string>
+    <string name="password_title">MQTT Password</string>
+    <string name="password_summary">User Password.%nCurrent value: %s</string>
```

Comment:
I'd prefer the password not being visible. Maybe just remove the second line with "current value"?

Reply:
You're right. It will be a good solution.